playbooks/slave_open_ports.yml

```diff
@@ -0,0 +1,11 @@
+  - name: Secure Slave
```

Comment:
I didn't notice that no actual ports are defined. Could we include this task in the slave security playbook but only execute if ports_whitelist is defined?